### PR TITLE
fix: Add Sriniketh J to the list

### LINKED ISSUE #855: Sriniketh J
srini047

### PR DESCRIPTION
**Pull request by srini047**
resolve #855

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1625,4 +1625,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/63718527/142635090-b0065905-edad-4d60-81d1-206ddf3093a4.png",
     "issueNumber": 853
   }
```
Comment (CodingSpecies) [inline data.json:1627]: Please add a comma after the brace, on line `1627` @srini047 .

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (srini047) [reply data.json]: I have done the necessary changes @CodingSpecies . Please re-review my PR.

File: data.json [hunk 2]
```diff
@@ -1624,5 +1624,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/63718527/142635090-b0065905-edad-4d60-81d1-206ddf3093a4.png",
     "issueNumber": 853
-  }
+  },
+  {
+  "name": "Sriniketh J",
+  "githubUsername": "srini047",
+  "imageUrl": "https://user-images.githubusercontent.com/81156510/142722621-fe15ceb1-3be2-4526-baea-024f5cdfd60a.png",
+  "issueNumber": 855
+}
```
Comment (krishguptadev) [inline data.json:1633]:
```suggestion
  }
```

File: data.json [hunk 3]
```diff
@@ -1624,5 +1624,11 @@
     "githubUsername": "JaideepGuntupalli",
     "imageUrl": "https://user-images.githubusercontent.com/63718527/142635090-b0065905-edad-4d60-81d1-206ddf3093a4.png",
     "issueNumber": 853
-  }
+  },
+  {
+  "name": "Sriniketh J",
+  "githubUsername": "srini047",
+  "imageUrl": "https://user-images.githubusercontent.com/81156510/142722621-fe15ceb1-3be2-4526-baea-024f5cdfd60a.png",
+  "issueNumber": 855
```
Comment (krishguptadev) [inline data.json:1632]:
```suggestion
    "issueNumber": 855
```

File: data.json [hunk 4]
```diff
@@ -1624,5 +1624,11 @@
     "githubUsername": "JaideepGuntupalli",
     "imageUrl": "https://user-images.githubusercontent.com/63718527/142635090-b0065905-edad-4d60-81d1-206ddf3093a4.png",
     "issueNumber": 853
-  }
+  },
+  {
+  "name": "Sriniketh J",
+  "githubUsername": "srini047",
+  "imageUrl": "https://user-images.githubusercontent.com/81156510/142722621-fe15ceb1-3be2-4526-baea-024f5cdfd60a.png",
```
Comment (krishguptadev) [inline data.json:1631]:
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/81156510/142722621-fe15ceb1-3be2-4526-baea-024f5cdfd60a.png",
```

File: data.json [hunk 5]
```diff
@@ -1624,5 +1624,11 @@
     "githubUsername": "JaideepGuntupalli",
     "imageUrl": "https://user-images.githubusercontent.com/63718527/142635090-b0065905-edad-4d60-81d1-206ddf3093a4.png",
     "issueNumber": 853
-  }
+  },
+  {
+  "name": "Sriniketh J",
+  "githubUsername": "srini047",
```
Comment (krishguptadev) [inline data.json:1630]:
```suggestion
    "githubUsername": "srini047",
```

File: data.json [hunk 6]
```diff
@@ -1624,5 +1624,11 @@
     "githubUsername": "JaideepGuntupalli",
     "imageUrl": "https://user-images.githubusercontent.com/63718527/142635090-b0065905-edad-4d60-81d1-206ddf3093a4.png",
     "issueNumber": 853
-  }
+  },
+  {
+  "name": "Sriniketh J",
```
Comment (krishguptadev) [inline data.json:1629]:
```suggestion
    "name": "Sriniketh J",
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Comment by srini047** [comment]
@krishguptadev Please review my PR.

**Review by krishguptadev — APPROVED** [review]

**Review by CodingSpecies — APPROVED** [review]
Thank you for all the changes @srini047

**Comment by srini047** [comment]
> Thank you for all the changes @srini047

It's my pleasure😊😊.

**Review by shreyaansjain06 — APPROVED** [review]
LGTM

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**